src/ReactiveDomain/ReactiveDomain.Tests/Logging/when_logging_disabled_commands_fired.cs

```diff
@@ -71,7 +65,7 @@ protected override void When()
         }
 
         [Fact(Skip = "pending deletion of log stream")]
-        public void can_verify_commands_not_logged()
+        public void can_verify_commands_are_not_logged()
```

Comment:
It would be nice to follow the convention of the other tests and make this "no_commands_are_logged"

This is true of all the tests that are still "can_verify..."